docs/src/getting_started/toolsinstallation.md

```diff
@@ -41,73 +45,172 @@ know to make the most of this shell:
 | `(reverse-i-search):` |            `^R`             |
 | `(forward-i-search):` |            `^S`             |
 
-- If you have a code editor installed on your development machine, you can go back and forth between the Julia session
-  and the editor by entering `julia> edit("<name of file>")`. You can jump right back into the Julia session once you're 
-  finished!
-- You can get a list of matches to a partially written word by entering the TAB key, which also serves to render an
-  Unicode symbol in the REPL (full list available in the [Unicode Input](https://docs.julialang.org/en/v1/manual/unicode-input/) section of the Julia manual): 
+* If you have a code editor installed on your development machine, you can go
+  back and forth between the Julia session and the editor by entering `julia>
+  edit("<name of file>")`. You can jump right back into the Julia session once
+  you're finished!
+* You can get a list of matches to a partially written word by entering the TAB
+  key, which also serves to render an Unicode symbol in the REPL (full list
+  available in the [Unicode
+  Input](https://docs.julialang.org/en/v1/manual/unicode-input/) section of the
+  Julia manual): 
   
   ```julia
   julia> pri[TAB]
   primitive type  print           println         printstyled
   julia> \pi[TAB] # renders π
   ```
-- You can run a standalone script by entering `include("<name of script>")` in Julian mode. For greater control, the 
-  equivalent of Python3's `if __name__ == __main__` condition would be `if abspath(<name of program file>) == @__FILE__`
-
-!!! warning
-    One common gotcha with the Julia REPL is that there are currently certain code changes that cannot be performed without 
-    restarting your session, such as redefining structs (see the example below). You can mitigate this by adding the 
-    [Revise](https://timholy.github.io/Revise.jl/stable/) package, although restarting the session is the simplest option.
+* You can run a standalone script by entering `include("<name of script>")` in
+  Julian mode. For greater control, the equivalent of Python3's `if __name__ ==
+  __main__` condition would be `if abspath(<name of program file>) == @__FILE__`
+
+!!! warning 
+    One common gotcha with the Julia REPL is that there are currently
+    certain code changes that cannot be performed without restarting your
+    session, such as redefining structs (see the example below). You can
+    mitigate this by adding the
+    [Revise](https://timholy.github.io/Revise.jl/stable/) package, although
```

Comment:
This phrasing is a bit misleading - Revise doesn't help with redefining structs...